src/aclients.c

```diff
@@ -221,6 +221,7 @@ int main (int argc, char *argv[])
 
 	if (num_clients < 1 || num_clients > MAX_CLIENTS) {
 	  printf ("Specify up to %d TNCs on the command line.\n", MAX_CLIENTS);
+	  dw_printf ("\e[0m\e\n\e[0J\e");
```

Comment:
It seems like `dw_printf ("\e[0m\e\n\e[0J\e");` always goes with `exit (x);`
Consider to combine those two. It might look like:
```
// Exit from DW and restore terminal color
void dw_exit(int exit_code)
{
    dw_printf ("\e[0m\e\n\e[0J\e");
    exit (exit_code);
}
```
```
dw_exit(1);
```